File: java/Baju.java
import java.util.List;

class Baju extends Aksesoris {
    private String untuk;
    private String size;
    private String merk;

    public Baju(int id, String namaProduk, double hargaProduk, int stokProduk, String jenis, String bahan, String warna, String untuk, String size, String merk) {
        super(id, namaProduk, hargaProduk, stokProduk, jenis, bahan, warna);
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    public String getUntuk() {
        return untuk;
    }

    public String getSize() {
        return size;
    }

    public String getMerk() {
        return merk;
    }

    public static List<String> getColumnHeaders() {
        List<String> headers = Aksesoris.getColumnHeaders();
        headers.add("UNTUK");
        headers.add("SIZE");
        headers.add("MERK");
        return headers;
    }

    public List<String> getData() {
        List<String> data = super.getData();
        data.add(untuk);
        data.add(size);
        data.add(merk);
        return data;
    }
}

File: python/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris):
    def __init__(self, id, nama, harga, stok, jenis, bahan, warna, untuk, size, merk):
        super().__init__(id, nama, harga, stok, jenis, bahan, warna)
        self.untuk = untuk
        self.size = size
        self.merk = merk

    def get_untuk(self):
        return self.untuk

    def get_size(self):
        return self.size

    def get_merk(self):
        return self.merk

    def get_column_headers(self):
        headers = super().get_column_headers()
        headers.extend(["UNTUK", "SIZE", "MERK"])
        return headers

    def get_data(self):
        data = super().get_data()
        data.extend([self.untuk, self.size, self.merk])
        return data